src/java/main/org/apache/zookeeper/ZooDefs.java

```diff
@@ -74,12 +74,16 @@
 
         public final int createTTL = 21;
 
+        public final int addPersistentWatch = 22;
+
         public final int auth = 100;
 
         public final int setWatches = 101;
 
         public final int sasl = 102;
 
+        public final int setWatches2 = 103;
```

Comment:
I don't think just because we are backporting this we need to keep this the same between the branches. I feel we should take advantage of the new version and keep the code as clean as possible. @hanm do you have an opinion on the matter?